Files aqsis-1.2.0.orig/.sconsign.dblite and aqsis-1.2.0/.sconsign.dblite differ
diff -Nru aqsis-1.2.0.orig/SConstruct aqsis-1.2.0/SConstruct
--- aqsis-1.2.0.orig/SConstruct	2007-02-18 23:25:17.000000000 +0100
+++ aqsis-1.2.0/SConstruct	2007-03-17 13:02:30.847697500 +0100
@@ -88,6 +88,9 @@
 	target_dir =  env.Dir('#/build_debug')
 opts.Add('install_prefix', 'Root folder under which to install Aqsis', defout)
 opts.Add('build_prefix', 'Root folder under which to build Aqsis', target_dir)
+opts.Add('pld_pluginsdir', 'Target directory for plugins')
+opts.Add('pld_shadersdir', 'Target directory for shaders')
+opts.Add('pld_configdir', 'Target directory for configuration file')
 
 # Make scons store build signatures in a single file _outside_ the install
 # hierarchy --- no more .sconsign files...
@@ -197,7 +200,7 @@
 #	env.AppendUnique(CPPPATH = [env.subst(a)] )
 env.AppendUnique(CPPPATH = ['$boost_include_path'])
 
-env.AppendUnique(CPPDEFINES=[('DEFAULT_PLUGIN_PATH', '\\"' + env.Dir('${PLUGINDIR}').abspath + '\\"')])
+env.AppendUnique(CPPDEFINES=[('DEFAULT_PLUGIN_PATH', '\\"' + env.Dir('$pld_pluginsdir').abspath + '\\"')])
 
 #
 # Enable MP dumping mode if specified
diff -Nru aqsis-1.2.0.orig/platform/default/SConscript aqsis-1.2.0/platform/default/SConscript
--- aqsis-1.2.0.orig/platform/default/SConscript	2007-02-04 11:44:55.000000000 +0100
+++ aqsis-1.2.0/platform/default/SConscript	2007-03-17 13:02:52.097025500 +0100
@@ -128,8 +128,8 @@
 
 # These absolute paths are necessary because there's no good general way of
 # getting a running program's install path on unix-like systems.
-env.AppendUnique(CPPDEFINES=[('DEFAULT_RC_PATH', '\\"' + env.Dir('$SYSCONFDIR').abspath + '\\"')])
-env.AppendUnique(CPPDEFINES=[('DEFAULT_SHADER_PATH', '\\"' + env.Dir('$SHADERDIR').abspath + '\\"')])
+env.AppendUnique(CPPDEFINES=[('DEFAULT_RC_PATH', '\\"' + env.Dir('$pld_configdir').abspath + '\\"')])
+env.AppendUnique(CPPDEFINES=[('DEFAULT_SHADER_PATH', '\\"' + env.Dir('$pld_shadersdir').abspath +'\\"')])
 env.Replace(SHLIB_VERSION_SUFFIX = '.%d.%d' %(version.major, version.minor))
 env.AppendUnique(SHLINKFLAGS=['-Wl,-soname=${TARGET.file}.%d' %version.major])
 
